Packs/CommonScripts/Scripts/CopyNotesToIncident/CopyNotesToIncident.py

```diff
@@ -27,8 +29,19 @@ def copy_notes_to_target_incident(args: Dict[str, Any]) -> CommandResults:
                 note_entries.append(entry)
 
         if len(note_entries) > 0:
-            demisto.executeCommand("addEntries", {"id": target_incident, "entries": note_entries})
-            md = f'## {len(note_entries)} notes copied'
+            result = []
+            retries = 3
+            sleep_time = 1
+            for i in range(retries):
+                result = demisto.executeCommand("addEntries", {"id": target_incident, "entries": note_entries})
+                if result:
+                    break
+                sleep(sleep_time)
+                sleep_time += 1
```

Comment:
use exponential backoff, i.e. `sleep_time *= 1`. Also, add a `debug` log to indicate that it failed

Reply:
removed sleep